docs/reference/3-admin-api/installation/admin-api-1x-for-odsapi-5x-6x/admin-api-1x-docker-installation.md

```diff
@@ -0,0 +1,184 @@
+# Admin API 1.x - Docker installation
+
+**Contents:**
+
+* [Before You Install](#before-you-install)
+  * [Compatibility & Supported ODS / API Versions](#compatibility-supported-ods-api-versions)
+* [Installation Instructions](#installation-instructions)
+  * [General Prerequisites](#general-prerequisites)
+* [Installation Instructions](#installation-instructions)
+  * [1\. Include Admin API in the ODS Docker Setup](#1-include-admin-api-in-the-ods-docker-setup)
+  * [2\. Relaunch the Docker Composition](#2-relaunch-the-docker-composition)
+  * [3\. Execute First-Time Configuration](#3-execute-first-time-configuration)
+
+# Before You Install
+
+This section provides general information you should review before installing the Ed-Fi ODS / API Admin API for v1.4.0.
+
+## Compatibility & Supported ODS / API Versions
+
+This version of the Admin API has been tested and can be installed for use with the Ed-Fi ODS / API v3.4 - 6.1. See the [Ed-Fi Technology Version Index](https://edfi.atlassian.net/wiki/spaces/ETKB/pages/20875717/Ed-Fi+Technology+Version+Index) for more details.
+
+# Installation Instructions
+
+## General Prerequisites
+
+The following are required to install the Admin API:
+
+* The Admin API provides an interface to administer an Ed-Fi ODS / API. Understandably, you must have an instance of the Ed-Fi ODS / API v3.4 - 6.1 deployed and operational before you can use the Admin API. Tested configurations include on-premises installation via [binary installation](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V520/pages/25100419/Getting+Started+-+Binary+Installation) or [source code installation](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V520/pages/25100348/Getting+Started+-+Source+Code+Installation).
```

Comment:
These pages have not been migrated yet. In a few weeks I will do a search for all Confluence links and try to fix any that should be changed.